Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/ScanAPIModelWizardPage.xaml.cs

```diff
@@ -284,7 +357,102 @@ public class SoapData
 
         private void IsSelected_FieldSelection_Click(object sender, RoutedEventArgs e)
         {
-            ObservableList<ApplicationAPIModel> CheckedList = GingerCore.General.ConvertListToObservableList(AddAPIModelWizard.AAMList.Where(x => x.IsSelected == true).ToList());
-        }        
+            ObservableList<ApplicationAPIModel> CheckedList = GingerCore.General.ConvertListToObservableList(AddAPIModelWizard.LearnedAPIModelsList.Where(x => x.IsSelected == true).ToList());
+        }
+
+        private void xCompareAndMergeButton_Click(object sender, RoutedEventArgs e)
+        {
+            ShowMergerWindow(sender);
+        }
+
+        void ShowMergerWindow(object sender)
+        {
+            DeltaAPIModel deltaAPI = null;
+            var fEl = sender as FrameworkElement;
+            if (fEl != null)
+                deltaAPI = fEl.DataContext as DeltaAPIModel;
+            if (deltaAPI != null)
+            {
+                bool showMergerWindow = false;
+                if (deltaAPI.DefaultOperationEnum == DeltaAPIModel.eHandlingOperations.MergeChanges)
+                    showMergerWindow = true;
+
+                if (mergerWindow == null)
+                    mergerWindow = new MergerWindow(deltaAPI);
```

Comment:
I think we should keep the MergingPage object also on the DeltaAPI object so we won't create new page each time so user will be able to continue editing from where he stopped last time he opened it per row